package.json

```diff
@@ -93,9 +92,7 @@
     "eslint-plugin-playwright": "^0.12.0",
     "happy-dom": "^8.1.5",
     "husky": "^8.0.3",
-    "jest": "^29.4.1",
     "lint-staged": "^13.1.2",
-    "msw": "^0.49.2",
```

Comment:
If you don't mind me asking what's msw?

Reply:
Looks like it is a mocking library ([msw](https://www.npmjs.com/package/msw))